lib/Notification/Notifier.php

```diff
@@ -385,7 +385,13 @@ protected function parseCall(INotification $notification, Room $room, IL10N $l)
 			} else {
 				$notification
 					->setParsedSubject($l->t('A group call has started'))
-					->setRichSubject($l->t('A group call has started'));
+					->setRichSubject($l->t('A group call has started'), [
+						'call' => [
+							'type' => 'call',
+							'id' => $room->getId(),
+							'call-type' => $this->getRoomType($room),
+						],
```

Comment:
It's missing the required name part:
https://github.com/nextcloud/server/blob/0913bc2157293acafa186afec4018ad25d7fbb22/lib/public/RichObjectStrings/Definitions.php#L170-L175
You can reuse the name from the activities:
https://github.com/nextcloud/spreed/blob/master/lib/Activity/Provider/Base.php#L143-L144

Reply:
:ping_pong: @mario